test/run

```diff
@@ -720,7 +723,7 @@ testYarn2WithoutYarnCache() {
 
 testYarn2WithoutWorkspaceModule() {
```

Comment:
Do we care about this test case/fixture given we've deleted all of our workspace plugin logic?

Reply:
for now, yes.  but once the feature flag is set to always be enabled we can remove it.